### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 13

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#339] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152325104-ed321093-befd-48c6-92d8-1084273f513d.png)

**Comment by Elia-Belli** [comment]
**1)** $f = f_{1,4} \land f_{2,4} \land f_{3,4}$
**3)** $f \in UNSAT$

**Comment by LuginiAndrea** [comment]
Scrivendole non in CNF, la seconda e la quarta:
2- S = {Insiemi con ripetizioni di { $1,2,3,4$ } di cardinalità 3}. Con $S_{i,j}$ si intende il j-esimo elemento dell'i-esimo elemento di $S$
$$f = \bigvee_{i=1}^{\textrm{Card}\left(S\right)}\bigwedge_{j=1}^3 f_{j, S_{i,j}}$$
4- Analogo, con S ={Insiemi senza ripetizioni di { $1,2,3,4$ } di cardinalità 3}
